### PR TITLE
Article throws error for an invalid route

### LINKED ISSUE #101: Tests for invalid links to assets in markdown files
We should test our markdown files for invalid links to other routes, and invalid links to image assets.

### PR DESCRIPTION
**Pull request by carpenterd777**
Closes #101 

I moved all of the routes to a separate file to be referenced using a `map`. This should be useful later when adding even more articles too.

### CODE REVIEW

**Comment by dyc3** [comment]
Also, we need unit tests for this

**Comment by carpenterd777** [comment]
Gotcha, I'll keep working on `Routes.js` and add unit tests for the valid route checking too

**Comment by dyc3** [comment]
The README article will also have to be updated.

**Comment by carpenterd777** [comment]
The biggest change that this branch now introduces is the utilization of `Router.js` for routes instead of dumping a bunch of routes in `App.js`. Now it should be easier to add new articles to the site.

File: sec.club/src/Router.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        articleProps: { source: "resources.md", title: "Resources",},
+        Component: lazy(() => import(/* webpackChunkName: "article" */ "./views/Article/Article.js")),
+    },
+    {
+        path: "/git-cheatsheet",
+        articleProps: { source: "git-cheatsheet.md", title: "Git Cheatsheet",},
+        Component: lazy(() => import(/* webpackChunkName: "article" */ "./views/Article/Article.js")),
+    },
+    {
+        path: "/dev-readme",
```
Comment (dyc3) [inline sec.club/src/Router.js]:
```suggestion
        path: "/dev/readme",
```

**Pull request merged**